python_programs/08_common_elements_between_two_list.py

```diff
@@ -0,0 +1,11 @@
+def common_elements(list1,list2):
```

Comment:
Looks fine. Try to optimise. Looks for a way, how can you reduce the usage of two for loop.

Reply:
You can merge without fixing it, but you should learn the optimized way.